src/parser/character/index.js

```diff
@@ -39,6 +39,16 @@ import { getSize } from "./size.js";
 import { getInitiative } from "./initiative.js";
 import { getCurrency } from "./currency.js";
 
+function getCharacterName(ddb) {
+  if (game.settings.get("ddb-importer", "character-update-policy-name")) {
+    return (ddb.character.name === "")
+    ? "Hero With No Name"
+    : ddb.character.name;
+  }
+  // because linting
+  return undefined;
```

Comment:
I'd rather parse out the name that's in the DDB JSON here, then replace it with the duplicate actors name - I try to build an accurate parse then do the changes afterwards

Reply:
yeah that makes sense, I wasn't 100% confident with the choice of doing this (or the latter one) and was expecting this to possibly be moved 😅